CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.1.1] - 2025/08/01
```

Comment:
Could you make a note in the changelog that the pipeline will no longer remove metadata values for samples where there was an issue calculating the earliest date?